grakn-engine/src/main/java/ai/grakn/engine/GraknEngineServer.java

```diff
@@ -112,15 +117,29 @@ public GraknEngineServer(GraknEngineConfig prop) {
         this.taskManager = startTaskManager(inMemoryQueue, redisCountStorage, jedisPool, lockProvider);
     }
 
-
     public static void main(String[] args) {
         GraknEngineConfig prop = GraknEngineConfig.create();
-        // Start Engine
+
+        // Start external components (Cassandra and Redis)
```

Comment:
@fppt do we want this? at one side it is good that people can still have the flexibility of doing it. But on the other side, this would expose the internal storage and queue which is what we're aiming to simplify.

Technically this is quite easy to do as I just need to add an option such as `cassandra_host=internal | <external cassandra IP address>`